Atomic/sign.py

```diff
@@ -0,0 +1,107 @@
+from . import util
+from . import Atomic
+import os
+import tempfile
+from .atomic import AtomicError
+import re
+
+
+class Sign(Atomic):
+    def sign(self):
+        # TODO
+        # Atomic is run as sudo. Should we work around that?
+
+        if self.args.debug:
+            util.write_out(str(self.args))
+
+        # Verify all the images provided are valid
+        self.check_input_validity()
```

Comment:
Resolved.  This is leftover from the previous assumption that the image would be pulled locally first.  Good catch.